### ISSUE TITLE
Is it possible to submit only the selected content to GPT, rather than the entire document?

### ISSUE THREAD

**Issue by Mingzefei** [bramses/chatgpt-md#90]
I think this feature is very important, especially when there is a lot of content in a document. 

It might have already been implemented, but I haven't found it.

**Comment by DeceptiveScorpion** [comment]
@Mingzefei yes, this is possible. 

To do this: 
1. You will need to disabled "Stream" 
2. Restart the plugin (disable and enabled) in the community plugins area.

- I verified using console that it's disabling steaming and the location of the .md does not impact the ability to disable streaming.



![image](https://github.com/bramses/chatgpt-md/assets/6787289/bf199e9b-f1f9-49ae-bc6f-8e3e7d5392e7)

**Comment by lukemt** [comment]
IMO If you disable streaming it just means that you get the responses slower, meaning: after the response has finished processing you get the whole response at once (but this takes longer)

but the requested feature is actually implemented in some way @Mingzefei you can see the discussion here: #50 and even more extensively here: #44

**Comment by Mingzefei** [comment]
@lukemt  Yes, the content you mentioned is what I want. I will close this issue and join the discussion. Thank you!
